misc/python/materialize/mzcompose/services.py

```diff
@@ -77,7 +78,10 @@ def __init__(
             *environment_extra,
         ]
 
-        command = ["--unsafe-mode"]
+        command = []
```

Comment:
@philip-stoev this is needed because the `sql-features/mzcompose.py` needs to start `Materialized` without this flag (otherwise all SQL-level feature flags will be always `on`).